src/App.js

```diff
@@ -3,15 +3,18 @@ import AddTodoForm from './AddTodoForm';
 import TodoList from './TodoList';
 
 function App() {
-  const [newTodo, setNewTodo] = React.useState(''); 
+  const [todoList, setTodoList] = React.useState([]);
+  const addTodo = (newTodo) => (
+    setTodoList([ ...todoList, newTodo ])
+  );
   return (
     <div style={{ textAlign: 'center' }}>
       <header>
         <h1>Todo List</h1>
         </header>
-        <AddTodoForm onAddTodo={setNewTodo}/>
-        <p>{newTodo}</p>
-        < TodoList />
+        <AddTodoForm onAddTodo={addTodo}/>
+
+        < TodoList todoList={todoList} />
```

Comment:
watch out with the space before `< TodoList`. Try this instead `<TodoList todoList={todoList} />`. Do you see the difference?